### DISCUSSION TITLE
It seems like Probable Location is less correct?

### DISCUSSION THREAD

**Discussion by xdTAGCLAN** [mxrch/GHunt#396] [General]
I tried to put my own gmail in Ghunt 1 and 2. And it seems that it is more accurate in Ghunt 1?
I don't know if this can be fixed but I just wanted to tell.

**Comment by mxrch** [comment]
Hi ! GHunt v1 was not taking the posted Photos in consideration, while GHunt v2 does (so it does now the Reviews + Photos).
So it may lower the probable location accuracy for your account, but it improves for the large majority.
